docs/react-native/index.md

```diff
@@ -16,6 +16,8 @@ This documentation is split into two sections:
 If you are just starting out with Embrace, follow the [**Integration Guide**](/react-native/integration/) to learn
 the key steps to successfully using our product.
 
+Also, if you are starting a fresh React Native project, notice that setting up full support for Embrace’s SDK can require additional configuration beyond the standard setup provided by `react-native init` or `@react-native-community/cli init` on the native side, mostly related to upgrading the minimum versions required for the SDK to run.
```

Comment:
I was thinking to put this warning in `docs/react-native/integration/index.md` under the "Requirements" section instead, maybe a callout modifying what you're saying here a bit to:
> Note that our minimum requirements on the native side tend to be a bit higher than those set by `react-native init` or `@react-native-community/cli init` by default for a fresh React Native project, please double-check that your project is configured correctly for the SDK to run

what do you think?